ptsemseg/models/pspnet.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, feature_scale=4, n_classes=21, is_deconv=True, in_channels=3,
         self.finaldeconvbnrelu1 = nn.Sequential(nn.ConvTranspose2d(filters[0], 32/feature_scale, 3, 2, 1),
                                       nn.BatchNorm2d(32/feature_scale),
                                       nn.ReLU(inplace=True),)
-        self.finalconvbnrelu2 = conv2DBatchNormRelu(32/feature_scale, 3, 32/feature_scale, 1, 1)
+        self.finalconvbnrelu2 = conv2DBatchNormRelu(in_channels=32/feature_scale, k_size=3, n_filters=32/feature_scale, padding=1, stride=1)
```

Comment:
Actually this implementation is just a replica of `linknet.py`, which I'd planned to finish, but didn't.

The real PSPNet ( which has dilated convolutions in the ResNet ) can be visualized [here](http://dgschwend.github.io/netscope/#/gist/841186c850a7ae49fd782111ebdb736f)

I'll merge this for now but just a cautionary statement -  this isn't the real PSPNet.